beacon_node/execution_layer/src/engine_api.rs

```diff
@@ -277,3 +303,20 @@ pub struct ProposeBlindedBlockResponse {
     pub latest_valid_hash: Option<Hash256>,
     pub validation_error: Option<String>,
 }
+
+// This name is work in progress, it could
+// change when this method is actually proposed
+// but I'm writing this as it has been described
+#[derive(Clone, Copy)]
+pub struct SupportedApis {
+    // I wonder if it would be better to use a bitmask here..
+    // It would be 8x more space efficient but I do remember
+    // Mac saying something about the performance being bad..
```

Comment:
```suggestion
```
Based on the convo between you and michael, seems like we can remove this comment?